source/guides/stacks/dbrs.rst

```diff
@@ -14,7 +14,7 @@ Overview
 ********
 
 The Database Reference Stack is integrated, highly-performant, open source,
-and optimized for 2nd generation Intel® Xeon® Scalable Processors and Intel®
+and optimized for 2nd generation Intel® Xeon® Scalable processors and Intel®
```

Comment:
```suggestion
and optimized for 2nd generation Intel® Xeon® Scalable processor family and Intel®
```

Reply:
I believe we are good to use 'processors' per the sample usage chart provided in the names DB (which shows an example of 'processor' pluralized).